rpc/tests/integration.rs

```diff
@@ -208,7 +208,7 @@ fn broadcast_tx_commit_null_data() {
 fn commit() {
     let response = endpoint::commit::Response::from_string(&read_json_fixture("commit")).unwrap();
     let header = response.signed_header.header;
-    assert_eq!(header.chain_id.as_ref(), EXAMPLE_CHAIN);
+    assert_eq!(header.chain_id.as_ref(), "dockerchain");
```

Comment:
Uh, I'm not sure, it was completely fine as it was. I don't think it was changed by me, it seems it was changed when the commit.json was regenerated. (The previous one had `cosmoshub-2` as the chain ID and the current one is `dockerchain`.) I don't mind it this way because most other assertions use constants to check validity. But we can revert to a const too.